aos/aos_milestones/IMpg/drive_mtaos.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<p>What would this line look like if we use <code>setttingsToApply</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>You mean if we use <code>salobj.set_summary_state</code>?</p><p> </p><p>If so, it would be:</p><p> </p><pre class="ql-syntax" spellcheck="false">await salobj.set_summary_state(mtcs.rem.mtaos, salobj.State.ENABLED, settingsToApply="impg")
</pre>